addons/test_mail/tests/test_mail_message.py

```diff
@@ -363,46 +360,45 @@ def test_mail_message_access_read_notification(self):
         self.message.write({'attachment_ids': [(4, attachment.id)]})
         self.message.write({'partner_ids': [(4, self.user_employee.partner_id.id)]})
         self.message.with_user(self.user_employee).read()
-        # Test: Bert has access to attachment, ok because they can read message
+        # Test: Employee has access to attachment, ok because they can read message
         attachment.with_user(self.user_employee).read(['name', 'datas'])
 
     def test_mail_message_access_read_author(self):
         self.message.write({'author_id': self.user_employee.partner_id.id})
         self.message.with_user(self.user_employee).read()
 
     def test_mail_message_access_read_doc(self):
-        self.message.write({'model': 'mail.channel', 'res_id': self.group_public.id})
-        # Test: Bert reads the message, ok because linked to a doc they are allowed to read
+        self.message.write({'model': 'mail.channel', 'res_id': self.test_channel_everyone.id})
+        # Test: Employee reads the message, ok because linked to a doc they are allowed to read
         self.message.with_user(self.user_employee).read()
 
     # --------------------------------------------------
     # CREATE
     # --------------------------------------------------
 
-    @mute_logger('odoo.addons.base.models.ir_model')
+    @mute_logger('odoo.addons.base.models')
```

Comment:
why the change?

Reply:
Becasue it's actually `odoo.addons.base.models.ir_rule` and  `odoo.addons.base.models.ir_model` producing the loggers. -> The public user not allowed to enter group channel. I will make it more clear with exactly those two.